src/components/common/StyledButton/index.js

```diff
@@ -51,6 +64,7 @@ function StyledButton(props) {
         disabled={props.disabled}
         onClick={props.onClick}
       >
+        {props.hasIcon ? <i className={props.iconClass}></i> : null}
```

Comment:
We won't be needing this props. If I dont pass *iconClass* , it will be undefined in the StyledButton component. So we can replace the condition like
>  props.iconClass ? <i className={props.iconClass}></i> : null